docs/getting-started/quickstart.rst

```diff
@@ -57,12 +57,24 @@ variable directly in your shell with:
     "Environment Variables" settings dialog, as explained e.g.
     `here <https://docs.python.org/3/using/windows.html#excursus-setting-environment-variables>`__.
 
-Finally start a notebook server by executing:
 
-.. code-block:: bash
+Jupyter
+-------
+Once you have activated your gammapy environment you can start
+a notebook server by executing::
 
     cd notebooks
     jupyter notebook
 
+
+Another option is to utilise the ipykernel functionality of Jupyter Notebook:
```

Comment:
Maybe this should specify that you need an environment with jupyter installed besides the gammapy one.

Reply:
Not sure what you mean, as this is just describing how to set up the kernel, which needs to be done in the gammapy environment?

Reply:
@Astro-Kirsty, something like that is better? 
```
An other option is to utilise the ipykernel functionality of Jupyter Notebook. It offers the possibility to choose a kernel from a pre-defined list.
In order to create kernels in such list, here are the command lines:
```
You can make some adaptation by making an inline proposal ;) thanks...